splunk_add_on_ucc_framework/templates/conf_files/inputs_conf.template

```diff
@@ -1,4 +1,7 @@
 {% for input_name in input_names %}
 {{ "[" ~ input_name ~ "]"}}
 python.version = python3
+{% if disabled and service_name == input_name%}
```

Comment:
I see there is this comparison. Wouldn't it be easier to edit input's rest handler and use `default` in the `RestField`?

```
    field.RestField(
        'disabled',
        required=False,
        validator=None,
        default={{ disabled }}
    )
```

Reply:
Adding it in the RH file would require the disabled property to be explicitly sent in the payload. Moreover, to enable or disable inputs, they have a dedicated endpoint which triggers the enabling and disabling of the endpoint, hence we'd require to set it in `inputs.conf` as to define the nature of newly created inputs.